File: testing/myorderfeature/app/src/main/java/com/example/myorderfeature/MainActivity.java
//Scenario 1: MainActivity for navigating to MyOrdersActivity
package com.example.myorderfeature;

import android.content.Intent;
import android.os.Bundle;
import android.widget.Button;
import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button startOrderButton = findViewById(R.id.btn_start_order);
        startOrderButton.setOnClickListener(v -> {
            // do nothing for now
        });


        Button cartButton = findViewById(R.id.btn_shopping_cart);
        cartButton.setOnClickListener(v -> {
            // do nothing for now
        });


        Button myOrdersButton = findViewById(R.id.btn_my_orders);
        myOrdersButton.setOnClickListener(v -> {
            Intent intent = new Intent(MainActivity.this, MyOrdersActivity.class);
            startActivity(intent);
        });
    }
}

File: testing/classicfeature/app/src/main/java/com/example/classicfeature/MainActivity.kt
package com.example.classicdrink

import android.os.Bundle
import android.widget.Button
import androidx.activity.ComponentActivity
import androidx.activity.compose.setContent
import androidx.activity.enableEdgeToEdge
import androidx.compose.foundation.layout.fillMaxSize
import androidx.compose.foundation.layout.padding
import androidx.compose.material3.Scaffold
import androidx.compose.material3.Text
import androidx.compose.runtime.Composable
import androidx.compose.ui.Modifier
import androidx.compose.ui.tooling.preview.Preview
import com.example.classicdrink.ui.theme.ClassicdrinkTheme
import android.content.Intent
import android.view.View
import android.widget.TextView

class MainActivity : ComponentActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.menulayout)
        enableEdgeToEdge()

        val thaiDrinkButton: Button = findViewById(R.id.thaiButton)
        val roseDrinkButton: Button = findViewById(R.id.roseButton)
        val matchaDrinkButton: Button = findViewById(R.id.matchaButton)

        val sharedClickListener = View.OnClickListener { v->
            if (v is Button){
                val text = v.text.toString()
                val intent = Intent(this, SecondActivity::class.java)
                intent.putExtra("drinkText", text)
                startActivity(intent)
            }
        }
        thaiDrinkButton.setOnClickListener(sharedClickListener)
        roseDrinkButton.setOnClickListener(sharedClickListener)
        matchaDrinkButton.setOnClickListener(sharedClickListener)
    }
}
        /*drinkButton.setOnClickListener {
            val buttonText = it as Button
            val text = buttonText.text.toString()

            drinkButton.setOnClickListener {
                val intent = Intent(this, SecondActivity::class.java)
                intent.putExtra("buttonText", text)
                startActivity(intent)
            }
        }*/

        /*val drinkButton: Button = findViewById(R.id.thaiButton)
        drinkButton.setOnClickListener {
            val intent = Intent(this, SecondActivity::class.java)
            intent.putExtra("buttonText", text)
        }*/


